x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/ranges/range_editor.tsx

```diff
@@ -0,0 +1,412 @@
+    defaultMessage: '\u2264',
+  });
+  const gteTooltipContent = i18n.translate(
+    'xpack.lens.indexPattern.ranges.greaterThanOrEqualTooltip',
+    {
+      defaultMessage: 'Greater than or equal to',
+    }
+  );
+  const ltPrependLabel = i18n.translate('xpack.lens.indexPattern.ranges.lessThanPrepend', {
+    defaultMessage: '\u003c',
+  });
+  const ltTooltipContent = i18n.translate('xpack.lens.indexPattern.ranges.lessThanTooltip', {
+    defaultMessage: 'Less than',
+  });
+
+  return (
+    <EuiPopover
+      ownFocus
+      isOpen={isPopoverOpen}
+      closePopover={() => {
+        setIsPopoverOpen(false);
+        if (isValidRange(tempRange)) {
+          // reset the temporary range for later use
+          setTempRange(range);
+          // send the range back to the main state
+          setRange(tempRange);
+        }
+      }}
+      button={<Button onClick={() => setIsPopoverOpen((open) => !open)} />}
+    >
+      <EuiForm>
+        <EuiFormRow>
+          <EuiFlexGroup gutterSize="s" responsive={false} alignItems="center">
+            <EuiFlexItem>
+              <EuiFieldNumber
+                value={isValidNumber(from) ? Number(from) : ''}
+                onChange={({ target }) =>
+                  setTempRange({
+                    ...tempRange,
+                    from: target.value ? Number(target.value) : -Infinity,
+                  })
+                }
+                append={
+                  <EuiToolTip content={gteTooltipContent}>
+                    <EuiText size="s">{gteAppendLabel}</EuiText>
+                  </EuiToolTip>
+                }
+                fullWidth={true}
+                compressed={true}
+                placeholder={FROM_PLACEHOLDER}
+                isInvalid={isValidNumber(from) && !isRangeWithin(tempRange)}
+              />
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiIcon type="sortRight" color="subdued" />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiFieldNumber
+                value={isFinite(to) ? Number(to) : ''}
+                onChange={({ target }) =>
+                  setTempRange({
+                    ...tempRange,
+                    to: target.value ? Number(target.value) : -Infinity,
+                  })
+                }
+                prepend={
+                  <EuiToolTip content={ltTooltipContent}>
+                    <EuiText size="s">{ltPrependLabel}</EuiText>
+                  </EuiToolTip>
+                }
+                fullWidth={true}
+                compressed={true}
+                placeholder={TO_PLACEHOLDER}
+                isInvalid={isValidNumber(to) && !isRangeWithin(tempRange)}
+              />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFormRow>
+        <EuiFormRow fullWidth>
+          <EuiFieldText
+            value={safeLabel}
```

Comment:
Please add `compressed` to make the label the same height as `EuiFieldNumber`

```suggestion
            compressed
            value={safeLabel}
```